build-tools/automation/yaml-templates/apk-instrumentation.yaml

```diff
@@ -19,7 +19,7 @@ steps:
       solution: ${{ parameters.project }}
       configuration: ${{ parameters.configuration }}
       msbuildArguments: >-
-        /restore
+        /restore /v:diag
```

Comment:
Did we need `/v:diag` here? It's already saving a `.binlog`.

Reply:
We should probably attach the binlog as a test attachment I think. At the moment if a test fails we have to download the entire set of test results as a zip , then find the failing test directory then look at the binlog.